libcontainer/seccomp/config.go

```diff
@@ -64,6 +64,7 @@ var archs = map[string]string{
 	"SCMP_ARCH_PPC":         "ppc",
 	"SCMP_ARCH_PPC64":       "ppc64",
 	"SCMP_ARCH_PPC64LE":     "ppc64le",
+	"SCMP_ARCH_RISCV64":     "riscv64",
```

Comment:
oh! think this one ended up in the wrong commit, or was it intentional to be part of the vendor update?

Reply:
Yes. Tied with the new version of libseccomp-golang.